jobs/kpi-forecasting/kpi_forecasting/models/prophet_forecast.py

```diff
@@ -116,3 +120,267 @@ def _predict_legacy(self) -> pd.DataFrame:
                 df[column] = 0.0
 
         return df[columns]
+
+    def _summarize(
```

Comment:
This is a lot to look through and I don't want to miss something important... is this just copy-pasted from the old version of the base class, or are there changes as well?

Reply:
Yeah everything is copied and pasted, the only changes are some of the stuff with the signatures we discussed above, and the `utcnow` thing

Reply:
Followed the directions here and just removed the tzinfo: https://blog.miguelgrinberg.com/post/it-s-time-for-a-change-datetime-utcnow-is-now-deprecated